src/ontology/mondo-edit.obo

```diff
@@ -202051,7 +202051,7 @@ xref: SCTID:768472004 {source="MONDO:equivalentTo"}
 xref: UMLS:C3891556 {source="NCBI:mim2gene_medline", source="MONDO:equivalentTo"}
 xref: UMLS:CN237731 {source="MONDO:equivalentTo"}
 is_a: MONDO:0000762 {source="DC-OMIM:300942"} ! syndrome caused by partial chromosomal duplication
-is_a: MONDO:0017009 {source="Orphanet:448372"} ! partial duplication of the short arm of chromosome X
```

Comment:
we should add partial duplication of the short arm of chromosome X as an excluded subclass, and add Orphanet:448372 as the source

Reply:
done! (I think :) )